### ISSUE TITLE
changefeedccl: add a metric for ranges waiting on the *client-side* limiter

### ISSUE THREAD

**Issue by asg0451** [cockroachdb/cockroach#136590]
The current "Ranges in Catchup mode" metric counts ranges performing catchup scans and ranges waiting on the replica-side limiter, but not ranges waiting on the client-side limiter. This leads to under-reporting. We should add the count of these to a new metric or to the existing one.

Jira issue: CRDB-45149

**Comment by asg0451** [comment]
closing as a duplicate of https://github.com/cockroachdb/cockroach/issues/134681
